src/components/v5/shared/SearchSelect/hooks.ts

```diff
@@ -12,12 +12,18 @@ export const useSearchSelect = (
         options: item.options.filter((option) => {
           const optionValue = option.value.replace('-', ' ');
           const optionWalletAddress = option.walletAddress || '';
+          const optionUserName = option.label || '';
 
           return (
             optionValue.toLowerCase().includes(searchValue.toLowerCase()) ||
             optionWalletAddress
               .toLowerCase()
-              .startsWith(searchValue?.toLowerCase() ?? '')
+              .startsWith(searchValue?.toLowerCase() ?? '') ||
+            (typeof optionUserName === 'string'
+              ? optionUserName
+                  .toLowerCase()
+                  .includes(searchValue?.toLowerCase())
+              : undefined)
```

Comment:
To be honest I am still a bit confused because `values` holds the same value as `walletAddress`. Why are we replacing hyphens in addresses and searching by them?